app/views/layout.mustache

```diff
@@ -5,12 +5,7 @@
 <html lang="en-us">
 <head>
   <script async src="https://www.googletagmanager.com/gtag/js?id=UA-26406144-13"></script>
-  <script>
-    window.dataLayer = window.dataLayer || [];
-    function gtag(){dataLayer.push(arguments);}
-    gtag('js', new Date());
-    gtag('config', 'UA-26406144-13');
-  </script>
+  <script src="{{{static_assets.gtag_js}}}"></script>
```

Comment:
hmm, I wonder if this could be async, it's not obvious to me...

Reply:
Good question. Because we tag the current timestamp in the `gtag('js', new Date())` I thought that it is tracking pageload times, and with that it is probably sync, but can change to async...

Reply:
I can't tell what it'll do in async mode...

If it'll actually work as intended?